core/src/tangram.h

```diff
@@ -11,12 +11,27 @@ namespace Tangram {
 
 class DataSource;
 
+enum LabelType {
+    icon,
+    text,
+};
+
 struct TouchItem {
     std::shared_ptr<Properties> properties;
     float position[2];
     float distance;
```

Comment:
Generic features don't have a fixed "position" that we could compare against during selection, so I'm not sure how we would come up with a "distance" even if we had a pick radius > 0.